### PR TITLE
785/improve faq table

### LINKED ISSUE #785: Improve operations table
See https://github.com/gnosis/cowswap/pull/760#issuecomment-862987776

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Closes #785 

Refactored FAQ table

## Desktop
screenshot_2021-06-18_12-54-34

## Mobile size
screenshot_2021-06-18_12-54-08

# Testing

Open FAQ and check contents

### CODE REVIEW

**Comment by biocom** [comment]
I like it, well done! 🎸  

Have some minor enhancements that I'd change
- Center align the table headers + content (except the first row).
- I'd use the emoji checkmark (strong green color..) ✅

**Review by biocom — APPROVED** [review]

File: src/custom/pages/Faq/index.tsx [hunk 1]
```diff
@@ -401,115 +382,93 @@ export default function Faq() {
...
+                  <td />
                 </tr>
               </tbody>
             </table>
           </div>
+          <ul>
+            <li>
+              <strong>Approve token</strong>:
+              <p>
+                Required step for being able to sell a token. Only needs to be done once. Afterwards, you will be able
```
Comment (W3stside) [inline src/custom/pages/Faq/index.tsx:444]:
```suggestion
                One-time-only, required step for being able to sell a token. Afterwards, you will be able
```

File: src/custom/pages/Faq/index.tsx [hunk 2]
```diff
@@ -401,115 +382,93 @@ export default function Faq() {
...
+                to trade the token using gasless transactions.
+              </p>
+            </li>
+            <li>
+              <strong>Wrap / Unwrap ETH</strong>:
+              <p>
+                Converts native ETH into an ERC20 compatible token: WETH. Only required if you need to sell ETH.
```
Comment (W3stside) [inline src/custom/pages/Faq/index.tsx:451]:
```suggestion
                Wrap: converts native ETH into its ERC20 compatible token form: WETH. Only required if you need to sell ETH.
```

File: src/custom/pages/Faq/index.tsx [hunk 3]
```diff
@@ -401,115 +382,93 @@ export default function Faq() {
...
+                Converts ERC20 compatible token WETH back to ETH. Only when you want to manually convert it to ETH.
```
Comment (W3stside) [inline src/custom/pages/Faq/index.tsx:452]:
```suggestion
                Unwrap: convert ERC20 compatible token WETH back into native ETH. Only when you want to manually convert it to ETH.
```

File: src/custom/pages/Faq/index.tsx [hunk 4]
```diff
@@ -401,115 +382,93 @@ export default function Faq() {
...
+                try to be executed using MEV protection against different on-chain liquidity sources or other CowSwap
+                users trading in the same block.
+              </p>
+            </li>
+            <li>
+              <strong>Cancel an order (Sign cancellation)</strong>:
+              <p>
+                Signature of a gasless off-chain cancellation request. This cancellation is considered
+                &ldquo;soft&rdquo; as it might not be placed with enough time for the solvers to take into
+                consideration. See more at <ContentLink href={'#can-i-cancel-an-order'}>this FAQ entry</ContentLink>.
```
Comment (W3stside) [inline src/custom/pages/Faq/index.tsx:468]:
```suggestion
                consideration. See more via <ContentLink href={'#can-i-cancel-an-order'}>this FAQ entry</ContentLink>.
```

**Comment by biocom** [comment]
@W3stside Committed your suggestions so we can get this PR going. Please re-review so we can merge.

File: src/custom/pages/Faq/index.tsx [hunk 5]
```diff
@@ -414,102 +405,88 @@ export default function Faq() {
...
+              <p>
+                <strong>Approve token</strong> <br />
+                One-time-only, required step for being able to sell a token. Afterwards, you will be able
+                to trade the token using gasless transactions.
+              </p>
+            </li>
+            <li>
+              <p>
+                <strong>Wrap ETH</strong> <br />
+                Wrap: convert native ETH into its ERC20 compatible token form: WETH. Only required if you need to sell ETH.
```
Comment (alfetopito) [inline src/custom/pages/Faq/index.tsx:463]:
```suggestion
                Convert native ETH into its ERC20 compatible token form: WETH. Only required if you need to sell ETH.
```

File: src/custom/pages/Faq/index.tsx [hunk 6]
```diff
@@ -414,102 +405,88 @@ export default function Faq() {
...
+              </p>
+            </li>
+
+            <li>
+              <p>
+                <strong>Unwrap ETH</strong> <br />
+                Unwrap: convert ERC20 compatible token WETH back into native ETH. Only when you want to manually convert it to ETH.
```
Comment (alfetopito) [inline src/custom/pages/Faq/index.tsx:470]:
```suggestion
                Convert ERC20 compatible token WETH back into native ETH. Only when you want to manually convert it to ETH.
```

**Review by anxolin — APPROVED** [review]

**Comment by anxolin** [comment]
@alfetopito can we merge this one? 

Maybe is even nice to merge to release/0.16 instead of develop?

**Comment by alfetopito** [comment]
> @alfetopito can we merge this one?
> 
> Maybe is even nice to merge to release/0.16 instead of develop?

I'll cherry pick into 0.16

**Comment by alfetopito** [comment]
On 0.16: 65478aba37ddc040d67f44e23e4c18d7870b1118

**Pull request merged**